### PR TITLE
Teachable Machine

### PR DESCRIPTION
**Pull request by Thiengen**
Added Teachable Machine

Co-Authored-By: StevenChang21 <82698910+StevenChang21@users.noreply.github.com>

### CODE REVIEW

**Comment by Thiengen** [comment]
Teachable Machine

**Comment by StevenChang21** [comment]
Woi, move ur code to a function and move the function to a new file. Then we can call those functions in sketch so code in sketch.js wouldn't be too long.

File: sketch.js [hunk 1]
```diff
@@ -1,22 +1,63 @@
...
+}
+
+function gotResults(error, results) {
+  if (error) {
+    console.error(error);
+    return;
+  }
+  label = results[0].label;
+  classifierVideo();
 }
```
Comment (StevenChang21) [inline sketch.js:60]: Can move these lines to other files? I wanna keep this file as short as possible

File: sketch.js [hunk 2]
```diff
@@ -1,22 +1,63 @@
 let columns_number, rows_number;
 let cell_length = 30;
+let label = " Waiting ..." ;
+let video ;
+let classifier ;
```
Comment (StevenChang21) [inline sketch.js]: And also these global variables, move to other file.

**Comment by StevenChang21** [comment]
Faster refactor, I want to merge ur PR by today. tq vry much @Thiengen

**Comment by Thiengen** [comment]
Done liao

File: sketch.js (hunk 1 above)
Comment (Thiengen) [reply sketch.js:60]: Ok

Comment (Thiengen) [reply sketch.js:60]: See What'sApp

**Comment by StevenChang21** [comment]
Next time give descriptive title and description on your commits. Tq

**Pull request merged**